src/main/java/com/documaster/rms/noark/ws/noarkentities/Dokumentversjon.java

```diff
@@ -28,8 +28,8 @@ public class Dokumentversjon extends NoarkEntityBase<Dokumentversjon> {
 	@JsonIgnore
 	public Long filstoerrelse;
 
-	@JsonIgnore
 	private String filnavn;
+	private boolean serializeFilnavn;
```

Comment:
Since we can't really set the field to null, I think a simple setter will do the job.

Reply:
Fixed w/ https://github.com/documaster/ws-client-n5-java/commit/10bb4a879b8379bd1f6a5a1df6c86499d9235861. I force-pushed, because the end result is much smaller than what 2 commit would have produced 😅